README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
+`--FitOption` | string | The [Fit Method](https://root.cern.ch/root/htmldoc/guides/users-guide/FittingHistograms.html#the-fit-method) to be used for fitting. Default is: R (Use the range specified in the function range)
+`--FitParamIGuess`  | Comma separated list of strings | Initial values for the parameters defined in the `Fit_Formula` field.  Note the order in which the parameters are given in the `Fit_Formula` field should match the order listed here. Explicitly for the `Fit_Formula = '[0]*x^2+[1]*x+[2]'` the initial guess of `4,5,6` would mean `[0] = 4`, `[1] = 5`, and `[2] = 6`.  Additionally, both numeric values and expressions formed from the above supported keywords can be used.  E.g. for a `Fit_Formula = 'gaus(0)'` the initial guess of `12.3, MEAN, 2.*SIGMA+23.5` can be assigned.
+`--FitRange` | Comma separated list of strings | Set the Fit range, default is 0,1000.
+`--FitPerform` | string| If true (default) perform a fit to the TObject defined in the [BEGIN\_PLOT] header this [BEGIN\_FIT] header is found in.
+`--FitLineSize` |float|Set the Fit Line Size, default is: 1 .
+`--FitLineStyle` |float|Set the Fit Line Style, default is: 1 .
+
+
+**Example of use:** 
+
+1) As with frameworkMain for each new shell navigate to the base directory of the repository and setup the environment via: 
+
+```
+source scripts/setup_CMS_GEM.sh
+```
+
+2) Let’s assume that you want to plot the first 60 lines of the second and third columns of the first sheet of the file QC3\_GE11-X-L-CERN-0002\_2017\_06\_01.xlsm which is stored in the base directory of the repository (../CMS\_GEM\_Analysis\_Framework/)
+
+What you have to do is:   
+
+```
+python2.7 python/Produce_Config_File.py —-file=QC3_GE11-X-L-CERN-0002_2017_06_01.xlsm —-SelectSheetNum=0 —-SelectColumnX=1 —-SelectColumnY=2 --SelectRowStart=1 --SelectRowEnd=61   
+```
+The options you have set are:
+						
+* `—-file=QC3_GE11-X-L-CERN-0002_2017_06_01.xlsm`	
+
+(the desired file stored in the base dir of the repository.
+If you want to plot the comparison of multiple files you can write:  
+
+`—-file=NameofFirstFile.xlsm —-file=NameofSecondFile.xlsm --file=NameofThirdFile.xlsm`),
+	
+	
+* `—-SelectSheetNum=0`
+
+	(the excel sheet number starting counting from zero),
+
+* `—-SelectColumnX=1 —-SelectColumnY=2`
+	
+	(The desired columns for read starting counting from zero),
+
+* `--SelectRowStart=1 --SelectRowEnd=61`
+
+	(The desired rows for read starting counting from zero).
+	
+After executing the command above, a **.cfg** file will be created in the base directory of the repository. If the user doesn't select the name of the **.cfg** file by adding the option **--OutputName=SomeName** in the command line above, the default name **Myconfig.cfg** will be set. In addition all the default values described in the options table above will be set.
```

Comment:
Filenames of scripts, pieces of code, classes, etc... in something that is a file or a code should be encapsulated, e.g. Myconfig.cfg should be `Myconfig.cfg` try to have input parameters or arguments be bolded.

Reply:
done..

Reply:
done..